packages/react/fast-components-react-base/src/data-grid/data-grid-cell.tsx

```diff
@@ -0,0 +1,176 @@
+
+    /**
+     * Renders the component
+     */
+    public render(): React.ReactNode {
+        if (
+            isNil(this.context) ||
+            isNil(this.context.dataGridProps) ||
+            isNil(this.props.columnDefinition)
+        ) {
+            return null;
+        }
+
+        const unhandledProps: React.HTMLAttributes<HTMLElement> = {
+            role: "gridcell",
+            tabIndex: this.isFocusCell() ? 0 : -1,
+            onKeyDown: this.handleKeyDown,
+            onFocus: this.handleFocus,
+        };
+
+        if (!isNil(this.props.columnDefinition.cell)) {
+            return this.props.columnDefinition.cell(
+                this.props,
+                this.generateClassNames(),
+                this.props.columnDefinition.columnDataKey,
+                this.rootElement,
+                this.focusTarget,
+                unhandledProps
+            );
+        } else {
+            return this.defaultCellRenderFunction(
+                this.props,
+                this.generateClassNames(),
+                this.props.columnDefinition.columnDataKey,
+                this.rootElement,
+                this.focusTarget,
+                unhandledProps
+            );
+        }
+    }
+
+    /**
+     * React life-cycle method
+     */
+    public componentDidMount(): void {
+        if (this.isDesiredFocusCell() && !isNil(this.rootElement.current)) {
+            this.rootElement.current.focus();
+        }
+    }
+
+    /**
+     * Generates class names
+     */
+    protected generateClassNames(): string {
+        return super.generateClassNames(
+            classNames(this.props.managedClasses.dataGridCell)
+        );
+    }
+
+    /**
+     * The default function that renders an unstyled content display
+     */
+    public defaultCellRenderFunction = (
+        props: DataGridCellProps,
+        className: string,
+        cellId: React.ReactText,
+        rootElement: React.RefObject<any>,
+        focusTarget: React.RefObject<any>,
+        unhandledProps: object
+    ): React.ReactNode => {
+        return (
+            <div
+                {...unhandledProps}
+                ref={rootElement}
+                data-cellid={cellId}
+                className={className}
+                style={{
+                    gridColumn: props.columnIndex,
+                }}
+            >
+                {props.rowData[props.columnDefinition.columnDataKey]}
+            </div>
+        );
+    };
+
+    /**
+     * returns true if this is the datagrid's current focus cell
+     */
+    private isDesiredFocusCell = (): boolean => {
+        if (this.context === null) {
+            return false;
+        }
+        return this.props.rowData[this.context.dataGridProps.dataRowKey] ===
+            this.context.desiredFocusRowKey &&
+            this.props.columnDefinition.columnDataKey ===
+                this.context.desiredFocusColumnKey
+            ? true
+            : false;
```

Comment:
this does not need to be a ternary, you can just use the conditional and it will be a boolean.